src/main/scala/ztis/recommender/Recommender.scala

```diff
@@ -0,0 +1,23 @@
+package ztis.recommender
+
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.slf4j.StrictLogging
+import akka.actor.{ActorSystem, Props}
+import akka.io.IO
+import spray.can.Http
+import akka.pattern.ask
+import akka.util.Timeout
+import scala.concurrent.duration._
+
+object Recommender extends App with StrictLogging {
+  val config = ConfigFactory.load("recommender")
+
+  implicit val system = ActorSystem("on-spray-can", config)
+
+  val service = system.actorOf(Props[RecommenderPortActor], "demo-service")
+
+  implicit val timeout = Timeout(5.seconds)
+  IO(Http) ? Http.Bind(service, interface = "localhost", port = 8080)
```

Comment:
I would extract port to configuration so it is possible to change it when starting the application, the same with the interface (or use "0.0.0.0")